www/wsgi_dashboards/webpack.config.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
```

Comment:
can we factorize this into buildbot_pkg?

Reply:
Agreed, WIP.

Reply:
Fixed now.